recipes/opl/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "opl" %}
+{% set version = "1.0.10" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "2ca4eb6c9ccf68de08617ea250db0e5e412b7ecbad8aabbc5008767758e45a31" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
```

This looks like pure Python, so you should be able to make this noarch:
```suggestion
  number: 0
  noarch: python
```